Frontend/Pages/Projects/BenefitsAndRisks/EqualitiesImpactAssessment.cshtml

```diff
@@ -5,11 +5,8 @@
 @{
     ViewBag.Title = (!ViewData.ModelState.IsValid ? "Error: " : "") + "Has an Equalities Impact Assessment been considered?";
     Layout = "_Layout";
-
-    var key = $"{nameof(Model.EqualitiesImpactAssessmentViewModel)}.{nameof(Model.EqualitiesImpactAssessmentViewModel.EqualitiesImpactAssessmentConsidered)}";
-    var formClasses = ViewData.ModelState.GetFieldValidationState(key) == ModelValidationState.Invalid 
-                        ? "govuk-form-group--error" 
-                        : "";
+    
+    var formClasses = ViewData.ModelState.IsValid ? "" : "govuk-form-group--error";
```

Comment:
I recommend using string.Empty here rather than an empty string literal - makes it clearer that this isn't a typo and it definitely is meant to be an empty string.

Reply:
Thanks @tim-wilde-nimble, have addressed this.